CSS/css_completions.py

```diff
@@ -62,7 +69,12 @@ def re_value(self):
         return re.compile(r"^(?:\s*(:)|([ \t]*))([^:]*)([;}])")
 
     @timing
-    def on_query_completions(self, view, prefix, locations):
+    def on_query_completions(
+        self,
+        view: sublime.View,
+        prefix: str,
+        locations: typing.List[sublime_types.Point]
+    ) -> typing.Union[None, typing.List[sublime_types.CompletionValue], typing.Tuple[typing.List[sublime_types.CompletionValue], sublime.AutoCompleteFlags], sublime.CompletionList]:
```

Comment:
I believe for this code this is enough.
```suggestion
    ) -> typing.Union[None, sublime.CompletionList]:
```